app/src/main/java/com/framgia/soundclound/screen/playtrack/PlayTrackActivity.java

```diff
@@ -27,10 +32,9 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         ActivityPlayTrackBinding activityPlayTrackBinding = DataBindingUtil.setContentView(
                 this , R.layout.activity_play_track);
-
+        mTrackRepository = new TrackRepository(TrackLocalDataSourceimpl.getInstance());
         PlayTrackViewModel mPlayTrackViewModel = new PlayTrackViewModel
-                (SharePreferences.getInstance().getListTrack(),
-                SharePreferences.getInstance().getTrack());
+                (mTrackRepository.getListTrackPlay(), mTrackRepository.getPositionTrack());
```

Comment:
ERROR: 'operator new lparen' have incorrect indentation level 16, expected level should be 8.